src/main/java/uk/co/bconline/ndelius/repository/db/ProbationAreaUserRepository.java

```diff
@@ -4,5 +4,8 @@
 
 import uk.co.bconline.ndelius.model.entity.ProbationAreaUserEntity;
 
+import java.util.List;
+
 public interface ProbationAreaUserRepository extends JpaRepository<ProbationAreaUserEntity, Long> {
+    List<ProbationAreaUserEntity> findProbationAreaUserEntitiesByUserUsername(String username);
```

Comment:
Could we use userId  instead here?  The username is case-insensitive so we'd need to make sure the check here is case-insensitive too, but using the userId is probably easier.